android/anc/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,7 @@
     <uses-permission android:name="android.permission.CAMERA" />
 
     <application
+        android:largeHeap="true"
```

Comment:
@ndegwamartin are you ok w/this change?

Reply:
@pld CQL Evaluation takes up a lot of memory, without this the app crushes

Reply:
I think its fine in this case since library we are using for CQL evaluation is the one taking up the memory leaves us no control over how to optimize the code. Usually we'd seek to refactor for optimization instead

Reply:
Yea makes sense, does having largeHeap true affect the system requirements for the devices the app can run on or anything like that?

Reply:
The heap size allocated to each app is dynamically calculated by the framework based on the the device specs, setting the flag to true nudges the system to allocate more space than would otherwise be allocated

Reply:
cool leaving this as is sounds good then